Form-Controls/index.html

```diff
@@ -14,8 +14,48 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <header> <h1>Product Pick</h1>
+    </header>
+    <main>
+      <div class="wrapper">
+        <form id="product-form">
+        <div>
+        <label for="name">Name</label>
+        <input type="text" id="name" name="name" placeholder="Your name should be 9-20 characters long">
+    </div>
+    <div>
+      <label for="email">Email</label>
+      <input type="email" id="email" name="email" placeholder="Enter your email address">
```

Comment:
How could you **require** this field be filled in?

Reply:
do I need to specify the characters in I need in my email box?